automation/check-patch.sh

```diff
@@ -209,7 +209,7 @@ run() {
         "ansible_modules_version=$ansible_modules_version"
         "kubevirt_openshift_version=$kubevirt_openshift_version"
         "openshift_playbook_path=$openshift_playbook_path"
-        "deploy_skydive=False"
+        "deploy_skydive=false"
```

Comment:
is this relevant to the patch?

Reply:
Boolean in ansible with lower characters

Reply:
I understand - but think that it belongs to a separate and quick bugfix PR.

Reply:
I also changed other parameters there so I think we can leave this change.